nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -75,6 +75,17 @@ RQ_QUEUES = {
 # For detailed configuration see: https://github.com/Suor/django-cacheops#setup
 CACHEOPS_REDIS = "redis://localhost:6379/1"
 
+# This is used for configuring Redis locks via django caching backend.
+CACHES = {
+    "default": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": "redis://127.0.0.1:6379/1",
```

Comment:
Do we need to use the IP address here, or can we change it to `localhost` to match the `CACHEOPS_REDIS` config above?

Reply:
@jathanism had some thoughts here

Reply:
My thought was that not only can we use localhost, but we could use the exact same URI for both `CACHEOPS_REDIS` and `CACHES["default"]["LOCATION"]`.

One thing to consider, however, is the possible confusion of `django-redis` cache settings vs. `django-cacheops` which does caching, but not using Django's cache framework. 

Another important consideration is the "what if" of using the same Redis instance/db for both and someone invalidates the Cacheops cache using `nautobot-server invalidate all` (which flushes the Redis db) and nukes a write lock in the process.